run-tests

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
+make dev
```

Comment:
This feels like it's going to interfere with the tests and at best combines concerns which ought to be separate. I'd really rather not do this.